HW_3/hw03.Rmd

```diff
@@ -0,0 +1,170 @@
+# Homework 03
+
+You will analyze data looking at the relationship between **green reputation** and three personality traits--**compassion**, **intellectual curiosity**, and **openness to experiences**.
+The dataset includes data from **students** and **non-students**.
+
+```{r}
+dictionary <- readr::read_csv("green_dictionary.csv")
+green_data <- readr::read_csv("green_data.csv")
+library(dplyr) 
+library(ggplot2)
+library(tidyverse)
+``` 
+
+For your assignment, do the following.
+
+  1. Inspect the item responses (e.g., with graphs or by summarizing distinct values). 
+     Is anything unusual?
+```{r}
+summary(green_data) 
+
+```
+  
+ 
+  2. Compute total scores for the four scales. 
+     Recode variables as needed.
+```{r}
+green_data %>%
+mutate(
+  across(eeverything(),
+ ~ recode(.x , "-99" = NA, .default = .x)
+))
```

Comment:
I think I figured out another reason why your values are weird at the end - you didn't save this as anything by naming a new object, so the recoded values didn't save to your next step.

Reply:
Thank you!!